WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -73,7 +73,15 @@ decisions = {
 	support_greens = {
 		only_playable = yes
 		potential = {
-			culture_group = latin
+			OR = {
+				culture_group = latin
+				culture = roman
+				has_landed_title = e_wre
+				has_landed_title = e_roman_empire
+				has_title = c_roma
+				has_title = b_rome
+				has_title b_roma
+			}
```

Comment:
I'll fix the trigger, but this isn't accurate. Only romance cultures will have this.